server.js

```diff
@@ -1,20 +1,79 @@
-var http = require('http');
-var secondLargest = require('./programs/secondLargest').secondLargest;
-var calculateFrequency = require('./programs/calculateFrequency').calculateFrequency;
-var flatten = require('./programs/flatten').flatten;
-var unflatten = require('./programs/unflatten').unflatten;
-const util = require('util');
+var http = require("http");
+var secondLargest = require("./programs/secondLargest").secondLargest;
+var calculateFrequency = require("./programs/calculateFrequency")
+  .calculateFrequency;
+var flatten = require("./programs/flatten").flatten;
+var unflatten = require("./programs/unflatten").unflatten;
+const util = require("util");
 
-var input_array = [21, 87, 91, 31, 97, 59, 8, 83, 27, 23, 76, 72, 31, 91, 65, 77, 84, 27, 56, 59];
+var input_array = [
```

Comment:
@abansal26 use input_array in one line.

Reply:
changes done as suggested by @sakodiya . Commit ID :- b9a0fc5ae6643c84c97b51c35a98edc691c3b8d8